src/modules/fancyzones/FancyZonesLib/trace.cpp

```diff
@@ -15,6 +15,9 @@
 #define EventSettingsKey "FancyZones_Settings"
 #define EventDesktopChangedKey "FancyZones_VirtualDesktopChanged"
 #define EventZoneWindowKeyUpKey "FancyZones_ZoneWindowKeyUp"
+#define EventMoveNewWindowIntoZone "FancyZones_MoveNewWindowIntoZone"
```

Comment:
Maybe it's better to name it `FancyZones_SnapNewWindowIntoZone` so it is consistent with `EventKeyboardSnapWindowToZone`